<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="info">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>No action was lunched!</string>
    </property>
   </widget>
   <widget class="QLabel" name="loading">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>571</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="scanbtn">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>8</y>
      <width>110</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SCAN</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="dup_table">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>100</y>
      <width>677</width>
      <height>460</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="midLineWidth">
     <number>4</number>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>4</number>
    </property>
    <property name="Size" stdset="0">
     <size>
      <width>-1</width>
      <height>-1</height>
     </size>
    </property>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Folder</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Size</string>
     </property>
    </column>
   </widget>
   <widget class="QDialog" name="kdialog">
    <property name="geometry">
     <rect>
      <x>2218</x>
      <y>431</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="filebrowse">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>84</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Browser</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="path_browser">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>290</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Location :</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="sort_combo">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>121</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Sort by</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Name (default)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Size</string>
     </property>
    </item>
   </widget>
   <zorder>loading</zorder>
   <zorder>info</zorder>
   <zorder>scanbtn</zorder>
   <zorder>dup_table</zorder>
   <zorder>kdialog</zorder>
   <zorder>filebrowse</zorder>
   <zorder>path_browser</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>sort_combo</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
